### ISSUE TITLE
How to Link Midjourney Account

### ISSUE THREAD

**Issue by bilskirnir** [imagineapi/imagineapi#48]
I had to change my discord password, and then I went to generate images later on with the api and received the error "Error creating thread L81: Unauthorized".

So I deleted my linked midjourney account thinking I can just easily add it again with the new password, now it won't let me re-add it. 

How can I re-add my midjourney account. Thank you
![addaccount](https://github.com/imagineapi/imagineapi/assets/5407036/af32a1b9-9c0b-4ddb-bfd5-cfda567e707d)

**Comment by kame345** [comment]
I am facing exactly the same problem.
I am starting to want to give up using this service because multiple issues are occurring.

**Comment by bilskirnir** [comment]
Yeah this should not be an issue, I am suprised the UI is this bad, if you can't re-add your midjourney account without trying to hassle the so far non existent customer service, then I am going to have to switch to https://www.mymidjourney.ai/, and I am going to ask Stripe for a refund.

**Comment by kame345** [comment]
Where is the form to cancel the subscription?
I searched the site but couldn't find it.

**Comment by bilskirnir** [comment]
There doesn't appear to be one?! 😬 Just like the rest of the crummy UI. I actually have an uncle who works as a compliance officer, and if I don't hear a response in the next few days I will foward this to him as fraud.

I think that is the reason he is ignorning us, the UI is so bad it doesn't have a way to reset your discord account, and he is too embarassed to admit it. I will also make sure I get refund and anyone else who needs it.

**Comment by William17Thorson** [comment]
Same problem
